website/docs/core-concepts/building-ui/designing-an-application/app-navigation.md

```diff
@@ -0,0 +1,87 @@
+# App Navigation
+
+We have introduced new navigation styles with better a default navigation - Top (Stacked), a variant for Top (Inline), and a collapsible Sidebar. You can now configure your app's navigation by opening the navigation settings tab inside the app settings pane and observe how your app with the selected navigation settings will look side by side as you change them.
```

Comment:
Some grammatical mistakes here!

Reply:
We should do a better job defining the diff b/w Top (Stacked), a variant for Top (Inline)

Reply:
For devs, its worth talking about the live preview mode as well!